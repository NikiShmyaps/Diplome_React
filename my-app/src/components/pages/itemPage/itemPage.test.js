import React from 'react';
import ItemPage from './itemPage';
import {shallow, mount} from 'enzyme';
import Service from "../../../services/service"

const itemPage = shallow(<ItemPage/>);

describe('Testing <MainPage/>', ()=> {
    it('ItemPage have rendered correctly', ()=>{
    expect(itemPage).toMatchSnapshot();
  })
  it('ItemPage state "items" is null', ()=>{
    expect(itemPage.state().items).toBeNull();
  })
});

// describe('Service test <ItemPage/>', ()=>{
//   const service = new Service();
//   const list = mount(<ItemPage
//                       getData={service.getCoffee()}
//                       renderItem={({name})=> name}/>)

//   it('check Service', ()=>{
//     list.setState({items: [{name: 'sfd', id: 1},{name: 'sfd', id: 2}]});
//   })
// }); 