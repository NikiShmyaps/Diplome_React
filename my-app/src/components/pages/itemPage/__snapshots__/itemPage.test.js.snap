// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <MainPage/> ItemPage have rendered correctly 1`] = `ShallowWrapper {}`;